netdeploy/remote/deployedNetwork.go

```diff
@@ -283,9 +338,424 @@ func (cfg DeployedNetwork) BuildNetworkFromTemplate(buildCfg BuildConfig, rootDi
 		return
 	}
 
+	if cfg.createBoostrappedNetwork {
+		fmt.Println("Generating db files ")
+
+		cfg.GenerateDatabaseFiles(cfg.BootstrappedNet, genesisFolder)
+	}
+
 	return
 }
 
+//GenerateDatabaseFiles generates database files according to the configurations
+func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, genesisFolder string) error {
+
+	fmt.Printf("starts %v\n", time.Now())
+	accounts := make(map[basics.Address]basics.AccountData)
+
+	genesis, err := bookkeeping.LoadGenesisFromFile(genesisFolder + "/genesis.json")
```

Comment:
avoid concatenating folder and file names directly. Use `filepath.Join` instead.